<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoundaryWidget</class>
 <widget class="QWidget" name="BoundaryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>640</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>19</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButtonIncongruous">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>异向</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSyntropy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>同向</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnRolbackNum">
   <property name="geometry">
    <rect>
     <x>937</x>
     <y>440</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>3</width>
     <height>480</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxEdge">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>75</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>边沿(绿)</string>
   </property>
   <widget class="QGroupBox" name="groupBoxEdgeDirection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>方向</string>
    </property>
    <widget class="QRadioButton" name="radioButtonEdgeBtoW">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>暗到亮</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonEdgeWtoB">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>亮到暗</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButtonEdgeX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>X方向</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonEdgeY">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>15</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Y方向</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>687</x>
     <y>450</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>回滚宽度:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxRoi">
   <property name="geometry">
    <rect>
     <x>671</x>
     <y>249</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>Roi(红)</string>
   </property>
   <widget class="QGroupBox" name="groupBoxRoiDirection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>94</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>方向</string>
    </property>
    <widget class="QRadioButton" name="radioButtonRoiBtoW">
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>暗到亮</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRoiWtoB">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>亮到暗</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>27</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButtonRoiX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>X方向</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRoiY">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Y方向</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>487</y>
     <width>1021</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>10</y>
       <width>631</width>
       <height>151</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>轮廓</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>211</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioButtonSkeletonFindBit1">
           <property name="text">
            <string>搜索区域:点1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelFind1">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>(100,100)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButtonSkeletonFindBit2">
           <property name="text">
            <string>搜索区域:点2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelFind2">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>(100,100)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>181</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButtonSkeletonStudyBit1">
           <property name="text">
            <string>边沿:点1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelStudy1">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>(100,100)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="radioButtonSkeletonStudyBit2">
           <property name="text">
            <string>边沿:点2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelStudy2">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>(100,100)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>181</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QRadioButton" name="radioButtonEdgeBit1">
           <property name="text">
            <string>ROI:点1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelEdgePoint1">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>(100,100)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QRadioButton" name="radioButtonEdgeBit2">
           <property name="text">
            <string>ROI:点2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelEdgePoint2">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>(100,100)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_16">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>341</width>
       <height>151</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>坐标微调</string>
     </property>
     <widget class="QPushButton" name="btnSkeletonDown">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSkeletonRight">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>→</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSkeletonLeft">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSkeletonUp">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>↑</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="txtRollBackWidth">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>446</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
